CenterForOpenScience/osf.io
# [PLAT-862] Storage usage for osfstorage

## Purpose

Let the user know how much storage they are using.

## Changes

- add property to Osfstorage NodeSettings that sums the size of each every version with test.
- minor frontend changes to make file size visible. 

## QA Notes

To observe these changes upload different files and different versions to the files tree. Reload the page. Their should be a number next to the OSF storage label that represents the storage usage of all files and all versions at that node. like the following:

<img width="1440" alt="screen shot 2018-06-05 at 4 54 58 pm" src="https://user-images.githubusercontent.com/9688518/41002254-38b575fe-68e1-11e8-869e-85f69027804c.png">



## Documentation

I'm unsure if this needs more context, it doesn't specify anything extra in the ticket.

## Side Effects

This feature is waffled under flag 'storage_usage'.

## Ticket

https://openscience.atlassian.net/projects/PLAT/issues/PLAT-862
